.github/workflows/build.yml

```diff
@@ -36,68 +39,60 @@ jobs:
       - name: Checkout Push to Registry action
         uses: actions/checkout@v4
 
-      - name: Install Just
-        shell: bash
-        run: |
-          set -eoux pipefail
-          JUST_VERSION=$(curl -L https://api.github.com/repos/casey/just/releases/latest | jq -r '.tag_name')
-          curl -sSLO https://github.com/casey/just/releases/download/${JUST_VERSION}/just-${JUST_VERSION}-x86_64-unknown-linux-musl.tar.gz
-          tar -zxvf just-${JUST_VERSION}-x86_64-unknown-linux-musl.tar.gz -C /tmp just
-          sudo mv /tmp/just /usr/local/bin/just
-          rm -f just-${JUST_VERSION}-x86_64-unknown-linux-musl.tar.gz
+      - name: Setup Just
+        uses: extractions/setup-just@v2
 
       - name: Check Just Syntax
         shell: bash
         run: |
           just check
 
-      - name: Image Name
-        shell: bash
-        run: |
-          IMAGE_NAME="$(just image_name)"
-          echo "IMAGE_NAME=${IMAGE_NAME}" >> $GITHUB_ENV
-
-      - name: Default Tag
-        shell: bash
-        run: |
-          DEFAULT_TAG="$(just generate-default-tag)"
-          echo "Default Tag: ${DEFAULT_TAG}"
-          echo "DEFAULT_TAG=${DEFAULT_TAG}" >> $GITHUB_ENV
-
       - name: Maximize build space
         uses: ublue-os/remove-unwanted-software@v7
         with:
           remove-codeql: true
 
-      - name: Build Image
-        id: build-image
-        shell: bash
-        run: |
-          just build
-
       - name: Generate tags
         id: generate-tags
         shell: bash
         run: |
-          alias_tags="$(just generate-build-tags \
-                     "latest" \
-                     "1" \
-                     "$(podman inspect ${{ env.IMAGE_NAME }}:${{ env.DEFAULT_TAG }} | jq -r '.[].Config.Labels["org.opencontainers.image.version"]')" \
-                     "${{ github.event_name }}" \
-                     "${{ github.event.number }}")"
-
-          echo "Tags for this Action..."
-          echo "$alias_tags"
-          echo "alias_tags=${alias_tags}" >> $GITHUB_OUTPUT
-
-      # Tag Images
-      - name: Tag Images
-        shell: bash
-        run: |
-          set -eoux pipefail
-          just tag-images "${{ env.IMAGE_NAME }}" \
-                          "${{ env.DEFAULT_TAG }}" \
-                          "${{ steps.generate-tags.outputs.alias_tags }}"
```

Comment:
Pulled this back into the github action workflow, since it's not needed for local dev.